source/vocab/items.ttl

```diff
@@ -210,7 +216,7 @@
 :itemUsed a owl:ObjectProperty ;
     rdfs:label "Item used"@en, "Exemplaret som använts"@sv ;
     rdfs:domain :ProvisionActivity ;
-    rdfs:range :Item .
+    rdfs:range :SingleItem .
```

Comment:
IIUC, sometimes it is unknown witch exact item was used; so this should probably stay as just `:Item`? Ping @ebengtsson

Reply:
Sounds reasonable